File: app/src/main/java/com/talktiva/pilot/request/RequestEvent.java
package com.talktiva.pilot.request;

import android.print.PrinterId;

import com.google.gson.annotations.SerializedName;
import com.talktiva.pilot.model.Invitation;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class RequestEvent implements Serializable {

    @SerializedName("eventId")
    private int eventId;

    @SerializedName("canInviteGuests")
    private Boolean canInviteGuests;

    @SerializedName("eventDate")
    private long eventDate;

    @SerializedName("invitations")
    private List<Invitation> invitations;

    @SerializedName("location")
    private String location;

    @SerializedName("private")
    private Boolean isPrivate;

    @SerializedName("title")
    private String title;

    public void setEventId(int eventId) {
        this.eventId = eventId;
    }

    public void setCanInviteGuests(Boolean canInviteGuests) {
        this.canInviteGuests = canInviteGuests;
    }

    public void setEventDate(long eventDate) {
        this.eventDate = eventDate;
    }

    public void setInvitations(List<Invitation> invitations) {
        this.invitations = invitations;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public void setIsPrivate(Boolean aPrivate) {
        isPrivate = aPrivate;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: app/src/main/java/com/talktiva/pilot/request/RequestEvent.kt
package com.talktiva.pilot.request

import com.google.gson.annotations.SerializedName
import com.talktiva.pilot.model.Invitation

import java.io.Serializable

class RequestEvent : Serializable {

    @SerializedName("eventId")
    var eventId: Int? = null

    @SerializedName("canInviteGuests")
    var canInviteGuests: Boolean? = null

    @SerializedName("eventDate")
    var eventDate: Long? = null

    @SerializedName("invitations")
    var invitations: List<Invitation>? = null

    @SerializedName("location")
    var location: String? = null

    @SerializedName("private")
    var isPrivate: Boolean? = null

    @SerializedName("title")
    var title: String? = null
}
